tests/unit/helpers/ember-prop-types-test.js

```diff
@@ -0,0 +1,285 @@
+/**
+ * Unit test for the frost-power-display component
+ */
+
+// Because we want to satisfy ESLint rules for not using references that were not previously defined...
+// ...AND... we need to be able to spy on the same methods we need to import the "mocha" library twice,
+// which is why the second import of it disables the ESLint warning about such duplication on a per-line basis
+import {expect} from 'chai'
+import Ember from 'ember'
+import {afterEach, beforeEach, describe, it} from 'mocha'
+import * as spyOnMocha from 'mocha' // eslint-disable-line no-duplicate-imports
+import sinon from 'sinon'
+import validatePropTypes, * as PropTypesHelper from 'dummy/tests/helpers/ember-prop-types'
```

Comment:
I beg to differ, I agree `forTestingOnly` is required, but I do not believe `PropTypesHelper` is required, since I've used this same mechanism before and never needed the second level of wrapping:
```js
sandboxy.spy(forTestingOnly, 'expect')
```
should work just fine, also you shouldn't need to store `spyExpect` you can just reference it directly after spying/stubbing it:

```js
expect(forTestingOnly.expect).to.have.callCount(1)
```